bindings/python/unicorn/unicorn.py

```diff
@@ -9,7 +9,7 @@
 import sys
 import weakref
 
-from . import x86_const, unicorn_const as uc
+from . import x86_const, arm64_const, unicorn_const as uc
```

Comment:
oh this is necessary. then this is OK, but we need to refactor the code to separate arch-dependent code into their own files later.